src/platform/graphics/constants.js

```diff
@@ -618,6 +618,133 @@ export const PIXELFORMAT_ATC_RGBA = 30;
  */
 export const PIXELFORMAT_BGRA8 = 31;
 
+/**
+ * 8-bit signed integer single-channel (R) format (WebGL2 only).
```

Comment:
I suggest changing `WebGL2 only` to `Not supported on WebGL1` on all these lines. As WebGPU supports those.